docs(review): review content (#1967)
scaleway/docs-content: dedibox-console/my-account/how-to/add-payment-method.mdx

  paragraph: This page explains how to add a payment method to your Dedibox console.
tags: dedibox dedibox-console payment billing
dates:
  validation: 2023-07-24
  posted: 2020-03-12
categories:
  - account
---

<Message type="requirement">
  - You have an account and are logged into the [Dedibox console](https://console.online.net) 
</Message>

Follow these steps to change the payment mode on your Scaleway Dedibox account:

> 1. Click your username, located next to "logged in as" in the top right corner of the console. From the pop-up menu, select **Billing**. The billing page displays.

Author: ```suggestion
1. Click your username, located next to "logged in as" in the top right corner of the console. From the drop-down menu, select **Billing**. The billing page displays.
```

> 1. Click your username, located next to "logged in as" in the top right corner of the console. From the pop-up menu, select **Billing**. The billing page will be displayed.

Reviewer: ```suggestion
1. Click your username, located next to "logged in as" in the top right corner of the console. From the pop-up menu, select **Billing**. The billing page displays.
```

> 1. Click your username, located next to "logged in as," in the top right corner of the console. From the pop-up menu, select **Billing**. The billing page will be displayed.

Author: ```suggestion
1. Click your username, located next to "logged in as" in the top right corner of the console. From the pop-up menu, select **Billing**. The billing page will be displayed.
```

> 1. Click on your username, located next to "logged in as," in the top right corner of the console. From the pop-up menu, select **Billing**. The billing page will be displayed.

Author: ```suggestion
1. Click your username, located next to "logged in as," in the top right corner of the console. From the pop-up menu, select **Billing**. The billing page will be displayed.
```

> 2. Click **Change payment mode** on the billing page. A list of available payment modes will appear. Choose the desired payment mode from the list, then click **Update payment mode**.

Reviewer: ```suggestion
2. Click **Change payment mode** on the billing page. A list of available payment modes appear. Choose the desired payment mode from the list, then click **Update payment mode**.
```
